### PR TITLE
xlfs and fix tfm

### PR DESCRIPTION
**Pull request by KevinRansom**
Updates the xlf files for a recent pr which didn't.

Also because I am an idiot I got a tfm wrong.

### CODE REVIEW

File: tests/FSharp.Compiler.Service.Tests/CSharp_Analysis/CSharp_Analysis.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard20</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```
Comment (cartermp) [inline tests/FSharp.Compiler.Service.Tests/CSharp_Analysis/CSharp_Analysis.csproj:3]: I think I've seen this change about 3 times now :)

Comment (KevinRansom) [reply tests/FSharp.Compiler.Service.Tests/CSharp_Analysis/CSharp_Analysis.csproj:3]: lol

Comment (KevinRansom) [reply tests/FSharp.Compiler.Service.Tests/CSharp_Analysis/CSharp_Analysis.csproj:3]: If I keep commenting, will you keep approving?

**Comment by abelbraaksma** [comment]
I thought that at some point in the past we had a test that checked if the xlf were updated correctly, which prevented "forgetting" such xlf changes. Maybe we should bring that test back?

**Pull request merged**